src/Appwrite/Event/Delete.php

```diff
@@ -54,6 +55,18 @@ public function setDatetime(string $datetime): self
         return $this;
     }
 
+    /**
+     * set Function
+     *
+     * @param Document $function
+     * @return self
+     */
+    public function setFunction(Document $function): self
+    {
+        $this->function = $function;
+        return $this;
+    }
```

Comment:
I dont think we need this. We can achieve the same by passing the function as a resource instead right ?

Reply:
Like we currently do here 
https://github.com/appwrite/appwrite/blob/master/app/workers/deletes.php#L49-L51

Reply:
Leftover probably. I cant see where we use it. Deleting